HM3CoreKill/HM3CoreKill/ck/HM3InGameTools.cpp

```diff
@@ -296,6 +358,14 @@ namespace ck
 		ImGui::SetNextWindowSize(ImVec2(500, 440), ImGuiCond_FirstUseEver);
 		if (ImGui::Begin("Actors viewer", pOpen))
 		{
+			if (ImGui::Button("Make some fun"))
```

Comment:
Test code. Better to add field "flags for the suit" for each actor and change mask in that place.